config/yaml/dffml_config_yaml/config.py

```diff
@@ -1,9 +1,8 @@
 """
 Description of what this config does
 """
-from typing import Dict
-
 import yaml
+from typing import Dict
```

Comment:
This file already follows the stdlib, then third party, then dffml style (yaml is third party)

So let's not change this file.